Examples/LambdaFunctions/serverless/HelloWorld-template.yml

```diff
@@ -0,0 +1,25 @@
+service: helloworld-swift-aws
+
+package:
+    artifact: .build/lambda/HelloWorld/lambda.zip
+
+provider:
+    name: aws
+    runtime: provided
+    iamRoleStatements:
+        - Effect: Allow
+          Action:
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+          Resource: "*"        
+
+functions:
+    hello:
+        handler: HelloWorld
+        memorySize: 128
+        events:
```

Comment:
other templates above do not include http integration. intentional?

Reply:
No, it wasn't intentional. I'll remove it.

Reply:
001b55a